<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>831</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>A</string>
     </attribute>
     <widget class="PlotWidget" name="plot_a_time">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="PlotWidget" name="plot_a_freq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>801</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>B</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>C</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>D</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_centralBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Muestreo de Señales</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="box_inputSignals">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Señal de Entrada</string>
     </property>
     <widget class="QComboBox" name="box_typeInputSignal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Senoidal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cuadrada</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audio</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_frecInputSignal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>90</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Frecuencia [Hz]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_frecInputSignal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>15000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_frecInputDuty">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duty Cycle [%]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_dutyInputSignal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QPushButton" name="import_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>125</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Import Audio [WAV]</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_samplesInputSignal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_samplesInputSignal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>142</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad de muestras: N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_paddingLength">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Padding Length</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_paddingLength">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="box_sampleHold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Tipo de Muestreo</string>
     </property>
     <widget class="QCheckBox" name="check_sampleHold">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>124</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample and Hold</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_dutyControlSignalSH">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_dutyControlSignalSH">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>15</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DC [%]</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_analogSwitch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>108</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Analog Switch</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_dutyControlSignalAS">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="box_filters">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtros</string>
     </property>
     <widget class="QCheckBox" name="check_FAA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>163</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FAA: Filtro Anti-Aliasing</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_FR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>150</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FR: Filtro Recuperdor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>15</y>
        <width>39</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>fp [Hz]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_freqFAA">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>40000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_freqFR">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>40000</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="box_controlSignal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Señal de Control</string>
     </property>
     <widget class="QLabel" name="label_frecControlSignal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>90</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Frecuencia [Hz]</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_frecControlSignal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>30000</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
